.github/workflows/run_examples_local.yml

```diff
@@ -1,4 +1,4 @@
-name: Run SPE1 demo
+name: Run examples local
```

Comment:
Why the more generic name?

Reply:
I see why now, because you have added both the SPE1 and the polynomial workspace to the same pipeline. This is however against the current idea of one workspace, one storage instance, so I believe we should move them back into two separate files...

Reply:
This was to separate the ones using sqlite and postgres. Since the osx/postgres corner of the ci matrix is not doable, it made this the most sense, unless we want to have one file for each cell in the matrix?

Reply:
Doing it like this, we do not have to run the installation process twice

Reply:
As an integration test it is useful to have multiple run on the same database